### PR TITLE
Add new action type

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java [hunk 1]
```diff
@@ -16,78 +16,129 @@
...
+        public Type getArgumentType()
+        {
+            return String.class;
+        }
     },
-    CLICK(WebElement.class)
+    SEND_KEYS
     {
+        private final Supplier<Type> argumentTypeSupplier = Suppliers
+                .memoize(() -> TypeUtils.parameterize(ArrayList.class, String.class));
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java]: just field instead of `Supplier`?

Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java]: why `ArrayList`, but not `List`?

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java [hunk 2]
```diff
@@ -16,78 +16,124 @@
...
         }
+
+        @Override
+        public Type getArgumentType()
+        {
+            return String.class;
+        }
     },
-    CLICK(WebElement.class)
+    SEND_KEYS
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java]: and we agreed on `PRESS_KEYS`?

**Comment by valfirst** [comment]
The message of last commit requires updates

**Comment by ikalinin1** [comment]
It seems like we missing integration tests for these steps/actions

**Comment by ikalinin1** [comment]
###  z

> It seems like we missing integration tests for these steps/actions

#230

**Pull request merged**